LINK/etc/default/evm

```diff
@@ -25,6 +25,9 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 # Location of certificates and provider keys
 export KEY_ROOT=/var/www/miq/vmdb/certs
 
+export APPLIANCE_REPO_DIRECTORY=/opt/manageiq/manageiq-appliance
```

Comment:
@Fryguy Not a fan of APPLIANCE_ROOT, I'm not sure what it means.

I'm not against renaming it but I think the current one describes exactly what it is, the directory containing the files of the appliance repo.